packages/tools/src/tools/annotation/ProbeTool.ts

```diff
@@ -608,7 +608,9 @@ class ProbeTool extends AnnotationTool {
         continue;
       }
 
-      const { dimensions, scalarData, imageData, metadata } = image;
+      const { dimensions, imageData, metadata } = image;
+      const scalarData =
+        'getScalarData' in image ? image.getScalarData() : image.scalarData;
```

Comment:
`image` is returned by `getTargetIdImage(...)`. This function returns `IImageData | CPUIImageData | IImageVolume` but only IImageData has `getScalarData()`.